ocaml-lsp-server/src/ocamlformat_rpc.mli

```diff
@@ -0,0 +1,7 @@
+type t
+
+val create : unit -> t
+
+val stop : t -> unit Fiber.t
+
+val query_type : t -> string -> (string, [> `Msg of string ]) result Fiber.t
```

Comment:
`string` should be labelled.

Reply:
I used a type alias to also make the return type clearer:

`val query_type : t -> type_ -> (type_, [> `Msg of string ]) result Fiber.t`